boxoffice/forms/item_collection.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+from baseframe import __
+import baseframe.forms as forms
+
+__all__ = ['ItemCollectionForm']
+
+
+class ItemCollectionForm(forms.Form):
+    title = forms.StringField(__("Item Collection title"),
+        validators=[forms.validators.DataRequired(__("Please specify a title")),
+        forms.validators.Length(max=250)], filters=[forms.filters.strip()])
+    description = forms.TinyMce4Field(__("Description"))
+    tax_type = forms.SelectField(u"Tax type", choices=[('GST', 'GST')], default="GST")
```

Comment:
Both choices are `GST`?

Reply:
It is just one option. `choices` needs to be tuple (value, label)

Reply:
`tax_type = forms.SelectField(__("Tax type"), choices=[('GST', 'GST')], default='GST')`

Reply:
Fixed